content/docs/user-guide/assets/pipeline/asset-cache.md

```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
+
 ### Asset Cache
 
- The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+
+The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
+
+In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
 
- The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
+If you delete or modify some files in your asset cache, the asset processor may not notice this because it assumes the only changes to the asset cache will be done by the asset processor. This includes deleting product asset files from your cache while the asset processor is not running.
 
- 
- In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+The Asset Cache is used during development, and by default the Editor and game launchers in development builds will load assets from the asset cache. Release builds are intended to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)

```

Comment:
Editorial suggestions. Is it more accurate to just say "release builds use bundled assets"?

```suggestion
The Asset Cache is used during development, and by default the Editor and game launchers in development builds load assets from the asset cache. Release builds use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```

Reply:
I mostly took this change. I wanted to make it clear it was the default behavior in release builds (implying you can change it).

See the updates here https://github.com/o3de/o3de.org/pull/1522